services/generic/barrier_control.md

```diff
@@ -1,36 +1,94 @@
-### Barrier control service
+# Barrier Control service
 
-The service represent devices like garage door openers, barriers, window protection shades, etc.
+The service represent devices like garage doors, barriers, window protection shades, and similar.
```

Comment:
Maybe we should also point out clearly the difference between this service and out lvl switch service. As device group that might use it is very similar. (?)

Reply:
Not sure what do you mean.

![image](https://user-images.githubusercontent.com/31538120/199433656-7f3a59bc-3f09-4e4a-a069-de2021bff1e6.png)

Even out_lvl_switch and out_bin_switch can be used in our app to operate blinds (or in fact garage door just without the fancy icon). It's just barrier control is more specialized service. All I think could be a note like this:

> Please note blinds and similar devices might also be supported in a limited way using more generic `out_bin_switch` and `out_lvl_switch services`. 

Is this what you meant?

Reply:
Yes this is what i mean...
Plus, I'm aware there are also Garage Gates which uses out_lvl_switch (e.g. Fibaro Nice Bidi-Zwave module).
In this case those out_lvl_switch-es are not possition aware. So can only start level - up/down and stop.
Z-Wave specification permits out level switches to be no position aware.

Maybe it just needs to be clearified - or added to out_lvl_switch description(?).